common.blocks/issue/issue.bemtree.js

```diff
@@ -40,15 +40,15 @@ block('issue').content()(function() {
             elem: 'content',
             content: issue.html
         },
-        this.data.url.pathname === '/' ? {
+        this.data.url.pathname.match(/^\/\d+\/$/g) !== null ? {
```

Comment:
Before this we have used block "Comments" in "Issue" block if url.pathname !== '/'. But we don't need comments(unfolded) on 404 page. 
Right now the unfolded "Comments" will be used only on Issue page, e.g /34/. 
If you have correct decision it will be great to see it :)